handlers/deploy.go

```diff
@@ -147,9 +149,10 @@ func makeSpec(request *requests.CreateFunctionRequest, maxRestarts uint64, resta
 				Delay:       &restartDelay,
 			},
 			ContainerSpec: &swarm.ContainerSpec{
-				Image:   request.Image,
-				Labels:  labels,
-				Secrets: secrets,
+				Image:    request.Image,
+				Labels:   labels,
+				Secrets:  secrets,
+				ReadOnly: request.ReadOnlyRootFilesystem,
```

Comment:
Something is definitely wrong with this implementation, I have tried deploying it with a local swarm,  have verified that that `request.ReadOnlyFilesystem` is `true` and that  but the resulting containers have `ReadonlyRootfs` set to `false` when I use `docker inspect`.   When I have it print the service spec, it is missing the  `ReadonlyRootfs` value as well.  Any additional eyes or help testing this would be really great

Reply:
Frustrating.. 😕 Is there anywhere else in the spec this needs to be added?

Can you add a print statement to confirm? How about checking the SHA/dev version in the logs too to check that the version is pushed properly?

Reply:
I have done this and according to other people that have tested, it seems to be getting the value and also working for them.  I suspect that one of my components might not be the version I suspect